specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -3257,6 +3257,54 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceBus/locations/{locationName}/namespaceOperations/{operationResultId}": {
```

Comment:
Is this an operationsStatus api? If so it should be called operationsStatus.
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#operation-resource-format